src/service/url-replacements-impl.js

```diff
@@ -451,7 +451,7 @@ export class UrlReplacements {
       return Promise.resolve();
     } else if (metric < 0) {
       // Metric is not yet available. Retry after a delay.
-      return loadPromise(this.win_).then(() => {
+      return loadPromise(this.win_.document).then(() => {
```

Comment:
I can use onDocumentReady if needed. The problem I am trying to solve here is that the promise that is returned by loadPromise(win) is never resolved. As a result, the flow that goes through this path never results in hits being sent from amp-analytics.  

I think performance-impl.js also has the same problem but I am not sure. Can revert that file's changes. 

Also, I can use onDocumentReady if that makes more sense. The main problem occurs [here](https://github.com/ampproject/amphtml/blob/master/src/event-helper.js#L97). Window does not have either of those properties and hence the function never returns true.